src/goMain.ts

```diff
@@ -97,7 +97,7 @@ export function activate(ctx: vscode.ExtensionContext): void {
 
 		offerToInstallTools();
 		if (checkLanguageServer()) {
-			const languageServerExperimentalFeatures = vscode.workspace.getConfiguration('go').get('languageServerExperimentalFeatures') || {};
+			const languageServerExperimentalFeatures: any = vscode.workspace.getConfiguration('go').get('languageServerExperimentalFeatures') || {};
```

Comment:
nit would be good to add a type for this if the output is specified anywhere

Reply:
See my previous comment [here](https://github.com/Microsoft/vscode-go/pull/2294#discussion_r253727770)